examples/flutter_gallery/lib/demo/cupertino/cupertino_navigation_demo.dart

```diff
@@ -236,17 +257,19 @@ class Tab1RowItem extends StatelessWidget {
               ),
               CupertinoButton(
                 padding: EdgeInsets.zero,
-                child: const Icon(CupertinoIcons.plus_circled,
+                child: const Icon(
+                  CupertinoIcons.plus_circled,
                   semanticLabel: 'Add',
                 ),
-                onPressed: () { },
+                onPressed: () {},
```

Comment:
this is one of those things I really don't care _that_ much, but my personal preference, especially if we have a formatter to take care of it for us, is that `{ }` be used for empty blocks, and `{}` be used for empty Maps and Sets.

Reply:
Easy to do. If you forget all your other comments I modify that immediately. :)